### PR TITLE
update lesson for postgres

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,57 +70,59 @@ public class Project {
...
-        student2.setDob(new Date());
-        student2.setStudentGroup(StudentGroup.ROSE);
+        student2.setName("Lee");
+        student2.setDob(LocalDate.of(1999,1,1));
+        student2.setStudentGroup(StudentGroup.DAISY);
+
+        // create a new student instance
+        Student student3 = new Student();
+        student3.setName("Amal");
+        student3.setDob(LocalDate.of(1980,1,1));
```
Comment (developerjaym) [inline README.md:117]: It looks like Student might still have a Date type Dob rather than LocalDate in one of the Student classes. (Line 318)

Comment (linda-seiter) [reply README.md:117]: fixed, thanks

**Review by developerjaym — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -621,3 +808,36 @@ public class JpaCreate {
 
 We have learned how to create one-to-many associations and how to keep the
 relationships in sync in the program by creating a list of related entities.
+
+A project is associated with exactly one student, thus the `Project` class stores
```
Comment (kcleland0818) [inline README.md:812]: Wonderful conclusion / summary! I think this really wraps up the point of this lesson and what these annotations do.

**Review by kcleland0818 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md:812]: thanks!

**Review by AlveeM — APPROVED** [review]

**Pull request merged**